### PR TITLE
Bumped version to 4.3. Added jpgraph stats developed by killerEye

### PR DESCRIPTION
**Pull request by randommonk**
- [x] Remove JpGraph (not used anymore)
- [x] Remove unused files, see https://github.com/sn3p/utstats/pull/5#issuecomment-355438743
- [ ] Update README

Maybe the number of js/css files isn't needed, I didn't really check skimming that down.
jpgraph is an external dependency available from http://jpgraph.net/ - It's added here for convenience but it can be removed from the repo and added with a description of how to add jpgraph to your php path

### CODE REVIEW

**Comment by sn3p** [comment]
Nice man 💯 A few questions:

1. Where do we use **Highcharts** and where do we use **JpGraph**? Are they somehow incorporated or doing different things?

2. Maybe we should try to separate dependancies/libs from custom code? E.g. create a **`lib/highcharts`** dir where we keep all the highcharts **js/css/img/themes/modules/adapters**/etc. I think this will be easier to maintain.

3. We could remove some unused files (for now):
   - [clickrows.js](https://github.com/randommonk/utstats/blob/3127ce0e642326ee2f2a293e0c1dd0ca0001800c/html/includes/js/clickrows.js) (not yet implemented)
   - [clocks.js](https://github.com/randommonk/utstats/blob/3127ce0e642326ee2f2a293e0c1dd0ca0001800c/html/includes/js/clocks.js) (unused, not sure what this does)
   - [jquery.tooltip.js](https://github.com/randommonk/utstats/blob/3127ce0e642326ee2f2a293e0c1dd0ca0001800c/html/includes/js/jquery.tooltip.js) (unused, using tooltipster instead)
   - `*.src.js` (unminified source files)

Don't mind helping with this, just let me know (don't want to hijack your PR).
Will do some more testing this weekend.

**Comment by sn3p** [comment]
> **kE**
Jpgraph is the old library that I used
Everything should be highcharts now, that is via Javascript
Yeah I'm sure, the charts in UT page are interactive

Would be nice if we can remove **JpGraph**!

**Comment by sn3p** [comment]
When Clearing the DB (Admin > Empty the database) it doesn't purge the `uts_chartdata` table.
This probably was already the case in the OW version? Could fix this in this PR or create an issue.

**Comment by sn3p** [comment]
I think this is ready to go after we update the README! 

And maybe a more extensive guide in the [Wiki](https://github.com/sn3p/utstats/wiki) on how to generate graphs for older games? I think kE has a script for that or something.

File: html/pages/match_info_server.php [hunk 1]
```diff
@@ -111,4 +115,15 @@
   </tr>
 </tbody></table>
 <br>';
+
+
+// Get Summary Info
+$teamscore = small_query("SELECT SUM(t0score + t1score + t2score + t3score) AS result FROM uts_match WHERE id = $mid");
```
Comment (sn3p) [inline html/pages/match_info_server.php]: Any particular reason why this is added here? An exact duplicate of these lines can be found earlier in this file:

https://github.com/randommonk/utstats/blob/8b2e0e3b3d3c127af48c6c3590794e625c2cff81/html/pages/match_info_server.php#L16-L22

Comment (randommonk) [reply html/pages/match_info_server.php]: Nope. Thought I commented here already actually but must be on the old pr. I'm not sure if these are even used at all. The duplication is just a merge mistake

**Pull request merged**